src/types/plugin-input.ts

```diff
@@ -11,12 +11,24 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
   ref: string;
 }
 
+function maxConcurrentTasks() {
+  return T.Transform(T.Record(T.String(), T.Integer(), { default: { member: 10, contributor: 2 } }))
+    .Decode((value) => {
+      // If admin is omitted, defaults to infinity
```

Comment:
Just remove everything admin related. Simplifies the code and it was out of scope in the first place.

Reply:
Possible roles are "admin", "member", "collaborator" + any other custom role added to the org. I can't remove admin logic, I could just hardcode a condition "if admin then infinity" which wouldn't change the code much I believe, because the admin case has to be handled.